AdminWebsite/AdminWebsite/ClientApp/src/app/services/recording-guard.service.ts

```diff
@@ -5,7 +5,7 @@ import { ParticipantModel } from '../common/model/participant.model';
     providedIn: 'root'
 })
 export class RecordingGuardService {
-    excludedCaseTypes: string[] = ['Court of Appeal Criminal Division'];
+    excludedCaseTypes: string[] = ['Court of Appeal Criminal Division', 'Crime Crown Court'];
```

Comment:
To avoid hard coding things like this, we should add a property to the case type to state whether or not it should be recorded, similar to the properties for hearing venues work allocation enabled and is Scottish

Reply:
I agree. Probably best to cover in a separate ticket